metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py

```diff
@@ -18,9 +18,9 @@
 assert AIRFLOW_PATCHED
 
 if TYPE_CHECKING:
-    from airflow import DAG
     from airflow.models import DagRun, TaskInstance
 
+    from airflow import DAG
     from datahub_airflow_plugin._airflow_shims import Operator
```

Comment:
Please revert unnecessary change

Reply:
I have executed `../gradlew :metadata-ingestion-modules:airflow-plugin:lintFix` and it has edited these files